### PR TITLE
fixed wrong link in docs

### PR DESCRIPTION
**Pull request by lwasser**

### CODE REVIEW

File: secrets/bootcamp-hub.yaml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+jupyterhub:
+  auth:
+    type: github
+    github:
+      clientID: 49802dd585160dac9ab8
```
Comment (lwasser) [inline secrets/bootcamp-hub.yaml]: it's not so secret. @betatim  i believe i'm missing an important secret step.

File: secrets/bootcamp-hub.yaml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+jupyterhub:
```
Comment (betatim) [inline secrets/bootcamp-hub.yaml]: let's remove this file from this PR

File: docs/source/day-to-day.rst [hunk 3]
```diff
@@ -82,7 +82,7 @@ Step two: Setup authentication
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Next decide how you'd like to authenticate your hub. You can use Github,
 Google or a "hash" based authenticator. Read more about that here
-`Read more about that here <https://earthlab-hub-ops.readthedocs.io/en/latest/>`_
+`Read more about that here <https://earthlab-hub-ops.readthedocs.io/en/latest/authentication.html>`_
```
Comment (betatim) [inline docs/source/day-to-day.rst]: We should find a way to do this without having to have the explicit URL here, something to fix in a new PR though, let's not hold this one up.

Comment (lwasser) [reply docs/source/day-to-day.rst]: cool... open to any ideas on that!

**Comment by betatim** [comment]
Merged!

**Pull request merged**